frontend/src/metabase/visualizations/components/TableSimple.jsx

```diff
@@ -80,15 +83,12 @@ export default class TableSimple extends Component {
   }
 
   componentDidUpdate() {
-    const headerHeight = ReactDOM.findDOMNode(
-      this.refs.header,
-    ).getBoundingClientRect().height;
-    const footerHeight = this.refs.footer
-      ? ReactDOM.findDOMNode(this.refs.footer).getBoundingClientRect().height
+    const headerHeight = this.headerRef.current.getBoundingClientRect().height;
+    const footerHeight = this.footerRef
```

Comment:
@daltojohnso I think this should be:
`const footerHeight = this.footerRef.current`, right?